lib/drivers/animasDriver.js

```diff
@@ -1779,6 +1954,7 @@ module.exports = function (config) {
         readSerialandModelNumber,
         readBasalProgramNames,
         readBasalPrograms,
+        readTimeAdjust
```

Comment:
yeah was just looking at the compare and the previous one did so thought I would double check